cdk.json

```diff
@@ -16,5 +16,6 @@
     ]
   },
   "context": {
+    "@aws-cdk/core:suppressTemplateIndentation": true
```

Comment:
This reduce the pipeline template ~200KB

https://github.com/aws/aws-cdk/discussions/22529